File: app/src/main/java/com/deepanjanxyz/notepad/domain/Note.java
package com.deepanjanxyz.notepad.domain;

public class Note {
    private long id;
    private String title;
    private String content;
    private String date;

    public Note(long id, String title, String content, String date) {
        this.id = id;
        this.title = title;
        this.content = content;
        this.date = date;
    }

    public long getId() {
        return id;
    }

    public String getTitle() {
        return title;
    }

    public String getContent() {
        return content;
    }

    public String getDate() {
        return date;
    }
}

File: domain/src/main/java/com/deepanjanxyz/notepad/domain/model/Note.kt
package com.deepanjanxyz.notepad.domain.model

data class Note(
    val id: Long = 0,
    val title: String = "",
    val content: String = "",
    val date: String = "",
    val isPinned: Boolean = false,
    val colorIndex: Int = 0,
    val tags: List<String> = emptyList(),
    val inTrash: Boolean = false,
    val inArchive: Boolean = false,
    val reminderTime: Long? = null
)
